src/components/AttachmentModal.tsx

```diff
@@ -540,6 +549,7 @@ function AttachmentModal({
                             {({safeAreaPaddingBottomStyle}) => (
                                 <Animated.View style={[StyleUtils.fade(confirmButtonFadeAnimation), safeAreaPaddingBottomStyle]}>
                                     <Button
+                                        ref={viewForwardedRef(submitRef)}
```

Comment:
Is there a reason why we use `viewForwardedRef` instead of  `viewRef`? The preferred approach seems to be `viewRef` as mentioned in our guidelines [here](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#refs)